### ISSUE TITLE
False positive PSR2.Methods.FunctionCallSignature.Indent

### ISSUE THREAD

**Issue by simPod** [squizlabs/PHP_CodeSniffer#2662]
Following code

```php
true
    ? null
    : array_map(
    function($e) {
        return $e;
    },
    []
);
```
should be formatted to
```php
true
    ? null
    : array_map(
        function($e) {
            return $e;
        },
        []
    );
```
but current sniff code tries to have it this way
```diff
     ? null
     : array_map(
         function($e) {
-            return $e;
+        return $e;
         },
         []
-    );
+);
```

**Comment by gsherwood** [comment]
I think this is actually the same issue as #2654, which will be fixed in 3.5.2. Are you able to test the current master code to confirm?

**Comment by simPod** [comment]
@gsherwood yes, it works on master. Thanks!

**Comment by gsherwood** [comment]
> yes, it works on master. Thanks!

Thanks a lot for testing that.
